config/locales/es.yml

```diff
@@ -0,0 +1,48 @@
+# Files in the config/locales directory are used for internationalization
+# and are automatically loaded by Rails. If you want to use locales other
+# than English, add the necessary files in this directory.
+#
+# To use the locales, use `I18n.t`:
+#
+#     I18n.t "hello"
+#
+# In views, this is aliased to just `t`:
+#
+#     <%= t("hello") %>
+#
+# To use a different locale, set it with `I18n.locale`:
+#
+#     I18n.locale = :es
+#
+# This would use the information in config/locales/es.yml.
+#
+# The following keys must be escaped otherwise they will not be retrieved by
+# the default I18n backend:
+#
+# true, false, on, off, yes, no
+#
+# Instead, surround them with single quotes.
+#
+# en:
+#   "true": "foo"
+#
+# To learn more, please read the Rails Internationalization guide
+# available at https://guides.rubyonrails.org/i18n.html.
+
+es:
+  sign_up:
+    account_creation_successful: "Account created successfully ( IN SPANISH )"
+  sign_in:
+    successful_sign_in: "Sign in successful ( IN SPANISH )"
+    wrong_email_password: "Wrong email or password ( IN SPANISH )"
+  sign_out:
+    signed_out: "Signed out ( IN SPANISH )"
+  access:
+    user_already_signed_in: "User has already signed in ( IN SPANISH )"
+    need_signup_for_access: "You need to be signed in to access that ( IN SPANISH )"
+    need_admin_for_access: "You need to be Admin to access that ( IN SPANISH )"
+  not_found:
+    bus_not_found: "Bus not found ( IN SPANISH )"
+    trip_not_found: "Trip not found ( IN SPANISH )"
+    boarding_point_not_found: "Boarding Point not found ( IN SPANISH )"
+    route_not_found: "Route not found ( IN SPANISH )"
```

Comment:
Look into the i18n-tasks gem to auto-generate translation!

Reply:
Thanks for the recommendation bhaia. Looked into i18n-tasks gem and studied how it helps to find and manage missing and unused translations.